biome.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.7.3/schema.json",
+  "organizeImports": {
+    "enabled": false
+  },
+  "vcs": {
+    "enabled": true,
+    "defaultBranch": "main",
+    "clientKind": "git",
+    "useIgnoreFile": true
+  },
+  "files": {
+    "include": ["**/*.js", "**/*.ts", "**/*.json", "**/*.jsonc", "**/*.jsx", "**/*.tsx"],
+    "ignoreUnknown": true,
+    "ignore": ["**/auto-imports.d.ts", "**/components.d.ts"]
+  },
+  "javascript": {
+    "formatter": {}
+  },
+  "formatter": {
```

Comment:
How about enable `"quoteStyle": "single"`? I think more frontend projects are using single quote

Reply:
Single quoteStyle adopted.

Half of the original codebase uses double and half uses single, so i adopted biome's default option before.